File: freecodecamp/ProjectEuler/Problem1_MultiplesOf3or5/problem-1-multiples-of-3-or-5.js
// Source: https://www.freecodecamp.org/learn/project-euler/project-euler-problems-1-to-100/problem-1-multiples-of-3-or-5

/* 
Description:
If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9, the sum of these multiples is 23.
Find the sum of all the multiples of 3 or 5 below the provided parameter value number 
*/

function multiplesOf3Or5(number) 
{
    let sum = 0;
    for (let i=0; i < number; i++)
    {
      if (i % 3 == 0 || i % 5 == 0)
      {
        sum += i;
      }
    }
    return sum;
  }
  
  module.exports = multiplesOf3Or5;
  

File: freecodecamp/ProjectEuler/Problem1_MultiplesOf3or5/problem-1-multiples-of-3-or-5.py
from typing import List

def multiples_of_3_or_5(number):
    sum: int = 0
    for i in range(number):
        if i % 3 == 0 or i % 5 == 0:
            sum += i
            # print(i)
    return sum

numbers: List[int] = [10, 49, 1_000, 8_456, 19_564]

for number in numbers:
    answer: int = multiples_of_3_or_5(number)
    print(f"Sum of all multiples of 3 and 5 below {number} is {answer}")
